libnavigation-ui/src/test/java/com/mapbox/navigation/ui/NavigationViewModelProgressObserverTest.java

```diff
@@ -10,25 +8,25 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 
-public class NavigationViewModelProgressChangeListenerTest {
+public class NavigationViewModelProgressObserverTest {
 
   @Test
   public void checksNavigationViewModelRouteProgressIsUpdatedWhenOnProgressChange() {
     NavigationViewModel mockedNavigationViewModel = mock(NavigationViewModel.class);
-    NavigationViewModelProgressChangeListener theNavigationViewModelProgressChangeListener =
-            new NavigationViewModelProgressChangeListener(mockedNavigationViewModel);
+    NavigationViewModelProgressObserver theNavigationViewModelProgressObserver =
+            new NavigationViewModelProgressObserver(mockedNavigationViewModel);
     RouteProgress theRouteProgress = mock(RouteProgress.class);
 
-    theNavigationViewModelProgressChangeListener.onRouteProgressChanged(theRouteProgress);
+    theNavigationViewModelProgressObserver.onRouteProgressChanged(theRouteProgress);
 
     verify(mockedNavigationViewModel).updateRouteProgress(eq(theRouteProgress));
   }
 
   /*@Test
```

Comment:
Why is this test commented out? 🤔

Reply:
I think it's commented when @RingerJK initially import the compatible SDK. This code is out of this PR's scope. It's impacted only because of the rename.
We could track it separately.

Reply:
Sure thing. Could you cut a ticket so we don't forget?

Reply:
this test is fixed and enabled now.